src/containers/qtreetbl.c

```diff
@@ -938,19 +964,30 @@ int node_check_rule5(qtreetbl_t *tbl, qtreetbl_obj_t *obj, int *path_len) {
 /**
  * Verifies that the (some) invariants of the red-black tree are satisfied.
  *
+ *  Property 1. Every node is either red or black.
+ *  Property 2. The root node is always black.
+ *  Property 3: Red nodes cannot have red children (no consecutive red nodes).
+ *  Property 4: Every leaf (null) node is considered black.
+ *  Property 5: Every path from the root of the tree to any leaf of the tree
+ *              has the same number of black nodes.
+ *  Property 6: A red link is allowed to the left child. (LLRB specific)
```

Comment:
I don't think it's worded correctly. Regular RB trees can have both left leaning and right leaning red links. In our case (2-3-4 LLRB trees) both left leaning red links and balanced 4 nodes (black node with 2 red links) are allowed.

Reply:
I like the updated comments you revised there